Sources/BraintreePayPalMessaging/BTPayPalMessagingLogoType.swift

```diff
@@ -15,7 +15,7 @@ public enum BTPayPalMessagingLogoType {
     case alternative
 
     /// "PayPal" as bold text inline with the message
-    case simple
+    case none
```

Comment:
Lets also confirm the docstring above is still accurate - it may be but just wanted to be sure

Reply:
Confirmed!! 👍